test/startservers.py

```diff
@@ -32,29 +32,18 @@ def run(self):
 processes = []
 
 
-def install(progs, race_detection):
-    cmd = "go install"
+def install(race_detection):
+    # Pass an empty BUILD_TIME flag to avoid constantly invalidating the build
+    # cache with new BUILD_TIMEs.
+    cmd = "make BUILD_TIME="
     if race_detection:
-        cmd = """go install -race"""
+        cmd = cmd + "GO_BUILD_FLAGS=-race"
```

Comment:
This looks like it would cause BUILD_TIME to be equal to GO_BUILD_FLAGS=-race? Need a space in here, at least. Maybe also some empty quotation marks for BUILD_TIME's value?